python/ray/tests/test_basic.py

```diff
@@ -1192,6 +1192,16 @@ def f():
         assert False
 
 
+def test_base_exception_raised(ray_start_shared_local_modes):
```

Comment:
add test for something inheriting from base

and double check we have tests if you raise the Keyboard & sys exit one